TCL/polarDensity_for_DTA.tcl

```diff
@@ -367,7 +375,7 @@ proc trajectory_leaflet_assignment {species headname tailname lipidbeads_selstr}
     global params
     set num_reassignments 0
     for {set update_frame $params(start_frame)} {$update_frame < $params(end_frame)} {incr update_frame $params(dt)} {
-        frame_leaflet_assignment $species $headname $tailname $lipidbeads_selstr $update_frame [expr $update_frame + $params(dt)] 
```

Comment:
The restrict_leaflet_sorter_to_Rmax param needs to be here (as opposed to inside the frame_leaflet_assignment proc)